loro-js/src/index.ts

```diff
@@ -179,10 +179,14 @@ export function isContainer(value: any): value is Container {
  */
 export function getType<T>(
   value: T,
-): T extends LoroText ? "Text"
```

Comment:
How do we maintain consistent lint？I found that the format of this part has been changed several times.

Reply:
eslint/prettier